lib/dslash_pack2.cu

```diff
@@ -2,17 +2,24 @@
 
 // STRIPED - spread the blocks throughout the workload to ensure we
 // work on all directions/dimensions simultanesouly to maximize NVLink saturation
-#define STRIPED
 // if not STRIPED then this means we assign one thread block per direction / dimension
 
+#ifdef NVSHMEM_COMMS
+#include <nvshmem.h>
+#include <nvshmemx.h>
+#else
+// MWTODO: how to handle striped ...
+#define STRIPED 1
+#endif
+
```

Comment:
so we don't have `STRIPED` support now when NVSHMEM is enabled?

Reply:
yes, at least for now. We can open an issue to track adding support.

Reply:
Tracked in #1084